sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -85,6 +85,10 @@
 --    arguments: none
 --    behavior: can take a long time.
 
+--  #IfUuidUpdatePatientsNeeded
```

Comment:
At the bottom of this script, add the following lines:
```
#IfUuidUpdatePatientsNeeded
#EndIf
```

Reply:
And at the bottom of the script above the lines above, create the following block:
```
#IfMissingColumn patient_data uuid
ALTER TABLE `patient_data` ADD `uuid` varchar(64);
(I was going to also add the unique keyhere , but realized that this will break since many if the uuids will be blank. Instead, in the sql upgrade custom function, after you are done creating the uuids, then check for existence of the unique key, and if it does not yet exist, then create it)
```